packages/block-library/src/column/edit.native.js

```diff
@@ -57,6 +79,32 @@ function ColumnEdit( {
 					value={ verticalAlignment }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Column settings' ) }>
+					<RangeControl
+						label={ __( 'Width' ) }
+						min={ 1 }
+						max={ 100 }
+						value={ columnWidths[ selectedColumnIndex ] }
+						onChange={ onWidthChange }
+						toFixed={ 1 }
+						rangePreview={
+							<ColumnsPreview
+								columnWidths={ columnWidths }
+								selectedColumnIndex={ selectedColumnIndex }
+							/>
+						}
+					/>
+				</PanelBody>
+				<PanelBody>
+					<FooterMessageControl
+						label={ __(
+							'Note: Column layout may vary between themes and screen sizes'
+						) }
+						textAlign="center"
```

Comment:
WDYT about setting `center` as a default one? We could remove it since we use only 'center' right now